### ISSUE TITLE
Some questions about gene co-expression matrices

### ISSUE THREAD

**Issue by zhaohangkai** [Yaolab-fantastic/TEMINET#1]
Why is the gene co-expression matrix used in the test set and the training set the same, can you tell us how this gene co-expression matrix is implemented, thanks for sharing!
